web/public/js/directives/user/cd-profile/view/template.dust

```diff
@@ -36,7 +36,7 @@
             <projects-list ng-if="!hideChampionProfileBlock('projects') && hasAccess(profile.userTypes, ['champion'])"></projects-list>
             <parents-list ng-if="!hideProfileBlock() && hasAccess(profile.userTypes, ['attendee-u13', 'attendee-o13'])"></parents-list>
             <children-list ng-if="loggedInUserIsCDFAdmin() || hasAccess(profile.userTypes, ['parent-guardian', 'mentor', 'champion'])"></children-list>
-            <programming-languages-list ng-if="!hideProfileBlock('programmingLanguages') && profile.widget.languages"></programming-languages-list>
+              <programming-languages-list ng-if="!hideProfileBlock('programmingLanguages') && profile.widget.languages"></programming-languages-list>
```

Comment:
Well, it's better to simply have a profile with badges, switch the parameter to hide the block to check the visibility, and one without badges at all :p